pkger/http_server.go

```diff
@@ -584,6 +625,21 @@ func pkgEncoding(contentType string) Encoding {
 	}
 }
 
+func convertEncoding(ct, rawURL string) Encoding {
```

Comment:
sorry, it's hard for me to tell but is this unit tested? it seems like a prime candidate for unit tests hitting all the branches in the switch statement

Reply:
not explicitly, its an internal implementation detail. Is tested in other tests that rely on this. It is purposeful that there is very limited white box testing in pkger.